addons/hr_payroll/static/src/xml/payslip_tree_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates>
+    <t t-name="PayslipListView.print_button">
+        <button type="button" class="btn btn-secondary o_button_print_payslip">
+            Print
+        </button>
+    </t>
+
+    <t t-extend="ListView.buttons" t-name="PayslipListView.buttons">
```

Comment:
Why this override ? It's not very modular btw.

Reply:
Maybe instead of this extend, render the template containing your button and add it directly to the element in the render_buttons method